CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 # Changelog
+
 This changelog keeps track of all changes to the packs-sdk. We follow conventions from [keepachangelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Unreleased
+
 - **Breaking Change** Added a validation rule that prevents the usage of varargsParameters for sync table getters
-which are not currently supported in the UI.
+  which are not currently supported in the UI.
+- **Future Breaking Change** The properties `id`, `primary`, and `featured` of object schemas will be renamed to `idProperty`, `primaryProperty`, and `featuredProperties`, respectively, to better clarify that their values refer to property names within the schema. During a migration period, you may use either name; the original names are now marked as deprecated and will be removed at a future date, no earlier than May 1, 2022.
```

Comment:
I still feel like we can improve on `primaryProperty`... Its only purpose is to set what the chip shows and what the display column is in a sync table, right? I think `displayProperty` might be better.

Reply:
Ya, I think `displayProperty` would be an improvement.

Reply:
cc @alicia-codaio who is thinking of future directions pack objects could be "displayed" - any concerns about this change?

Reply:
hm. `displayProperty` seems okay but maybe too vague for me to feel super confident about futureproofing. based on those two main areas (what's visible on the chip, and the primary column on the sync table) what about something like `titleProperty`, where it feels more definitively singular (whereas you might "display" more than one property)?

Reply:
@ekoleda-codaio thoughts on `titleProperty` vs `displayProperty`? I will defer to you on this weighty decision 😄 Both are reasonable options to me.

Reply:
I named the section of the docs ["Display value"](https://coda.io/packs/build/latest/guides/advanced/schemas/#display-value), so I'm biased towards `displayProperty`. To me "title" implies something more formal, and for some schemas the displayed value could be as simple as number. If I get the tie breaker, then put me down for `displayProperty`.

Reply:
`displayProperty` is probably the most intuitive. Dunno about future-proofing against hypotheticals, but if we think we might allow multiple properties to feed into that, doesn't seem inconceivable we'd add `displayProperties` at that point, or change the type also allow an array.